STATUS: INLINE SUCCESSFUL
MODIFIED: de.strullerbaumann.visualee.dependency.boundary.DependencyContainerTest.testGetRelevantClassesDirectlyConnected
MODIFIED FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/dependency/boundary/DependencyContainerTest.java
CLEANER: de.strullerbaumann.visualee.dependency.boundary.TestGroup100Case0.test257
CLEANER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/dependency/boundary/TestGroup100Case0.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.SourcecodeFilterTest.testIsOkIncludeTrue
POLLUTER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/SourcecodeFilterTest.java
ORIGINAL CLEANER SIZE: 19
NEW CLEANER SIZE: 2
ELAPSED TIME: 62.22000002861023
==========================
@@ -91,0 +91,4 @@
+ {
+     de.strullerbaumann.visualee.filter.boundary.FilterContainer filterContainer0 = de.strullerbaumann.visualee.filter.boundary.FilterContainer.getInstance();
+     filterContainer0.clear();
+ }
